airflow/api_connexion/parameters.py

```diff
@@ -48,6 +45,27 @@ def format_datetime(value: str):
         )
 
 
+def check_limit(value: int):
+    """
+    This checks the limit passed to view and raises BadRequest if
+    limit exceed user configured value
+    """
+    max_val = int(conf.get("api", "maximum_page_limit"))
```

Comment:
Can you add this option to reference docs(airflow/config_templates/config.yml file) and docs (/docs directory)?

Reply:
I added it to config.yml but can't find where to add it in docs.

Reply:
We don't have a specific page, so we have to create. My title suggestion: "Using the REST API".  It will be similar to CLI. For CLI, we have [reference docs](https://airflow.readthedocs.io/en/latest/cli-ref.html) and [topic guide](https://airflow.readthedocs.io/en/latest/usage-cli.html).

Reply:
Here is what I did. https://github.com/apache/airflow/pull/9431/commits/7e231ca7e48617c5c1af718ee4c134370aba211e

and this too https://github.com/apache/airflow/pull/9431/commits/861f33d5c86ec5777ea2084dab96e4919140d817